openslides/core/static/js/core/base.js

```diff
@@ -100,6 +101,13 @@ angular.module('OpenSlidesApp.core', [
             // Loads all projector data
             Projector.findAll();
 
+            // Loads all chat messages data and their user_ids
+            ChatMessage.findAll().then( function(chatmessages) {
```

Comment:
I added a permission check:
 `if (operator.hasPerms("core.can_use_chat")) ...`
But I get this error if I add `operator` to `loadGlobalData`:
"Circular dependency found: loadGlobalData <- operator <- loadGlobalData"
@ostcar How can we fix this?